robocop/utils/misc.py

```diff
@@ -18,7 +18,7 @@
 from robocop.rules import RuleSeverity
 from robocop.exceptions import InvalidExternalCheckerError
 
-IS_RF4 = VERSION.startswith('4')
+IS_RF4 = VERSION.startswith('4')  # FIXME: We need better version matching - for 5.0.0
 DISABLED_IN_4 = frozenset(('nested-for-loop', 'invalid-comment'))
 ENABLED_IN_4 = frozenset(('if-can-be-used', 'else-not-upper-case', 'variable-should-left-aligned'))
```

Comment:
BTW Not related but I think we need to change this rule name from `variable-should-left-aligned` to `variable-should-be-left-aligned`

Reply:
Right, it's not backward compatible change though - I will create issue for it.